<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReagentConsumeItem</class>
 <widget class="QWidget" name="ReagentConsumeItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>715</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>217</width>
    <height>715</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>217</width>
    <height>715</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_up" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>217</width>
     <height>63</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f2f7fb;

    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>122</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lab_title">
      <property name="styleSheet">
       <string notr="true">width: 127px;
height: 25px;
font-family: PingFang SC;
font-weight: bold;
font-size: 26px;
color: #333333;
line-height: 35px;
border : 0px;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>122</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_mid" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>63</y>
     <width>217</width>
     <height>247</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f2f7fb;
border-radius : 0px;
</string>
   </property>
   <widget class="QPushButton" name="btn_foreground_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>87</y>
      <width>196</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #37adba;

border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lab_hint">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;

border: 0px;
width: 46px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #000000;
line-height: 35px;</string>
    </property>
    <property name="text">
     <string>提示信息</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_background_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>196</width>
      <height>230</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>196</width>
      <height>230</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>196</width>
      <height>230</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;

border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;

border : 1px solid #37adba;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>btn_background_color</zorder>
   <zorder>btn_foreground_color</zorder>
   <zorder>lab_hint</zorder>
  </widget>
  <widget class="QWidget" name="widget_down" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>217</width>
     <height>404</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">background-color: #f2f7fb;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;</string>
      </property>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>vv </string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
